float64 speed
# in meters per second, center of robot
float64 turnRadius
# in meters, 0 = infinite (i.e. straight), negative = left turn, positive = right turn
float64 distance
# as an arcLength in meters

# Q: how should "turn in place" and "strafe" be defined? waiting to see how manual control handles problem @Lutu
